index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="intro">
+        <header id="header">
+            <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+            <nav id="navbar">
+                <ul>
+                    <li class="content-padding"><a href="#">Portfolio</a></li>
+                    <li class="content-padding"><a href="#">About</a></li>
+                    <li class="content-padding"><a href="#">Contact</a></li>
+                </ul>
+                <a href="#" class="content-padding" id="menu"><img src="images/menu-icon.png" alt="menu"></a>
+            </nav>
+
+        </header>
+        <section id="headline">
```

Comment:
Kindly fix your background image in moble version by resizing it, or downloading the one in the figma template. Currently, its only visible on desktop but the mobile version has no backround image
Your design:
<img width="560" alt="Screenshot 2022-09-23 at 12 34 11" src="https://user-images.githubusercontent.com/48546992/191933676-c019a9c6-d6ac-4964-823c-fab99163ea35.png">

Figma template
<img width="905" alt="Screenshot 2022-09-23 at 12 35 53" src="https://user-images.githubusercontent.com/48546992/191933790-c5b9d24c-a1c0-4b1f-a71a-7690da41f434.png">